### PR TITLE
Create 02.md

### PR DESCRIPTION
**Pull request by goodboychan**
Added korean translation of 02.md

### CODE REVIEW

**Comment by skyil7** [comment]
Week 2 -> 2주차
Lecture part A -> 이론 part A
Practicum -> 실습

위와 같이 변경해주시면 감사하겠습니다. 수고하셨습니다!

또한, part A에서 section을 세션이라 번역하셨는데, part나 단락 등으로 수정함이 적절해 보입니다.
검토 부탁드립니다.

백지오 드림

**Comment by goodboychan** [comment]
말씀하신 내용 반영했습니다.
감사합니다.

**Comment by Atcold** [comment]
I'm confused. This is *not* ready to go, correct?

File: docs/ko/week02/02-1.md [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+
+
+
+- Scalar-valued function
+
+  <img src="{{site.baseurl}}/images/week02/02-1/scalar-valued.PNG" alt="scalar-valued" style="zoom:50%;" />
+
+    - Used to represent cost functions
+    - Has an implicit scalar output
+    - Takes multiple inputs and outputs a single value (usually the distance between the inputs)
```
Comment (Atcold) [inline docs/ko/week02/02-1.md]: There's a lot of English, still.

**Comment by skyil7** [comment]
Yeap.

**Comment by skyil7** [comment]
I mean it's good to go. I reviewed it.

File: docs/ko/week02/02-1.md (hunk 1 above)
Comment (skyil7) [reply docs/ko/week02/02-1.md]: A ha. I got it.
It was just 02.md for this pr. And he is working on 02-1 now.

**Comment by Atcold** [comment]
Okay, merging `02.md`.

**Comment by Atcold** [comment]
I've removed `02-1.md` from the current branch and updated `_config.yml`.

File: docs/ko/week02/02-1.md (hunk 1 above)
Comment (goodboychan) [reply docs/ko/week02/02-1.md]: Ah, Sorry for confusing. I should work it locally.

**Pull request merged**